template/src/components/header/__tests__/Navbar.test.tsx

```diff
@@ -3,10 +3,15 @@
  */
 import { render, screen } from '@testing-library/react';
```

Comment:
is `__tests__` the right approach?
It feels a bit too much Python for TS taste.

In the follow up PR, let's move the file back into the main `header` folder.